File: 2199-two-furthest-houses-with-different-colors/two-furthest-houses-with-different-colors.java
class Solution {
    public int maxDistance(int[] colors) {
        int max = 0;
        for(int i=0;i<colors.length/2;i++){
            for(int j=1;j<colors.length;j++){
                if (colors[i] != colors[j]){
                    if (max < j-i){
                        max = j-i;
                    }
                }
            }
        }
        return max;
    }
}

File: 2199-two-furthest-houses-with-different-colors/two-furthest-houses-with-different-colors.py
class Solution:
    def maxDistance(self, colors: List[int]) -> int:
        max = 0
        for i in range(len(colors)//2):
            for j in range(1,len(colors)):
                if colors[i] != colors[j]:
                    if max < j-i:
                        max = j-i
        return max